test/test_additional_importer.py

```diff
@@ -618,3 +618,103 @@ def test_import_report_skipped(single_import_parameters):
             concepts=0,
         )
     }
+
+
+def test_import_report_add_concept(single_import_parameters):
+    dataset, original, excel, concept_name = single_import_parameters
+    c_c_id = dataset["ParameterTable", "id"].name
+    c_c_name = dataset["ParameterTable", "name"].name
+    concepts = {c[c_c_name]: c[c_c_id] for c in dataset["ParameterTable"]}
+    sheet = MockSingleExcelSheet(
+        [
+            [
+                "English",
+                "Form",
+                "phonemic",
+                "orthographic",
+                "Segments",
+                "procedural_comment",
+                "Comment",
+                "Source",
+                "phonetic",
+                "variants",
+            ],
+            [
+                "one",
+                "form",
+                "phonemic",
+                "orthographic",
+                "f o r m",
+                "-",
+                "None",
+                "source[10]",
+                "phonetic",
+                "",
+            ],
+        ]
+    )
+    sheet.title = "new_language"
+
+    # Import this single form in a new language
+    assert read_single_excel_sheet(
+        dataset=dataset,
+        sheet=sheet,
+        entries_to_concepts=concepts,
+        concept_column=concept_name,
+    ) == {
+        "new_language": ImportLanguageReport(
+            is_new_language=True, new=1, existing=0, skipped=0, concepts=0
+        )
+    }
+
+    # Import it again, with a new concept
+    sheet = MockSingleExcelSheet(
+        [
+            [
+                "English",
+                "Form",
+                "phonemic",
+                "orthographic",
+                "Segments",
+                "procedural_comment",
+                "Comment",
+                "Source",
+                "phonetic",
+                "variants",
+            ],
+            [
+                "three",
+                "form",
+                "phonemic",
+                "orthographic",
+                "f o r m",
+                "-",
+                "None",
+                "source[10]",
+                "phonetic",
+                "",
+            ],
+        ]
+    )
+    sheet.title = "new_language"
+
+    assert read_single_excel_sheet(
+        dataset=dataset,
+        sheet=sheet,
+        entries_to_concepts=concepts,
+        concept_column=concept_name,
+    ) == {
+        "new_language": ImportLanguageReport(
+            # TODO: Actually, this isn't a new language. The difference between
+            # adding forms for a language that is not in the LanguageTable yet,
+            # but already has forms in the FormTable, and adding something
+            # completely new, is washed out by read_single_language. The
+            # interpretation of “Does this language still need to be added to
+            # the LanguageTable?” for is_new_language is consistent.
+            is_new_language=True,
+            new=0,
+            existing=1,
```

Comment:
I think it should be existing=0 and concepts=1

Reply:
I'll check. There was a complex merge that I may not have paid enough attention to, so this may have slipped by.